collectors/ebpf.plugin/ebpf.c

```diff
@@ -533,6 +535,36 @@ void ebpf_print_help() {
  *
  *****************************************************************/
 
+/**
+ * Start Ptherad Variable
+ *
+ * This function starts all pthread variables.
+ *
+ * @return It returns 0 on success and -1.
+ */
+int ebpf_start_pthread_variables()
+{
+    if (pthread_mutex_init(&lock, NULL)) {
```

Comment:
BTW
> pthread_mutex_init always returns 0

https://manned.org/pthread_mutex_init.3#head6

Reply:
I removed them, this is so common that I wrote the check in the automatic.